packages/@aws-cdk/aws-backup/lib/plan.ts

```diff
@@ -0,0 +1,256 @@
+ * what rules will the plan will follow.
+ */
+export class BackupPlan extends BackupPlanBase {
+  /**
+   * Imports a backup plan from its attributes.
+   */
+  public static fromBackupPlanAttributes(scope: Construct, id: string, attrs: BackupPlanAttributes): IBackupPlan {
+    class Import extends BackupPlanBase {
+      public readonly backupPlanArn: string = attrs.backupPlanArn;
+      public readonly backupPlanId: string = attrs.backupPlanId;
+      public readonly backupPlanVersionId: string = attrs.backupPlanVersionId;
+      public readonly backupPlanName: string = attrs.backupPlanName;
+      public readonly vault: IBackupVault = attrs.vault;
+    }
+
+    return new Import(scope, id);
+  }
+  /**
+   * A string representatino of the backup plan ARN
+   */
+  public readonly backupPlanArn: string;
+  /**
+   * The backup plan Id.
+   */
+  public readonly backupPlanId: string;
+  /**
+   * The backup version Id.
+   */
+  public readonly backupPlanVersionId: string;
+  /**
+   * The backup vault to use for all backups.
+   */
+  public readonly vault: IBackupVault;
+  /**
+   * A list of rules that specify the rules of the plan.
+   */
+  public rules: PlanRule[];
+  /**
+   * The backup plan name.
+   */
+  public readonly backupPlanName: string;
+
+  constructor(scope: Construct, id: string, props: BackupPlanProps) {
+    super(scope, id);
+
+    const resource = new CfnBackupPlan(this, 'Resource', {
+      backupPlan: {
+        backupPlanName: props.backupPlanName,
+        backupPlanRule: Lazy.anyValue({produce: () => this.renderPlanRules(this.rules)}),
+      },
+    });
+
+    this.backupPlanName = props.backupPlanName;
+    this.backupPlanArn = this.getResourceArnAttribute(resource.attrBackupPlanArn, {
+      service: 'backup',
+      resource: 'backup-plan',
+      resourceName: resource.ref,
+    });
+    this.backupPlanId = this.getResourceNameAttribute(resource.ref);
+    this.backupPlanVersionId = resource.attrVersionId;
+
+    this.vault = props.vault;
+    this.rules = props.rules;
+  }
+
+  /**
+   * Adds the given plan rules.
+   */
+  public addPlanRule(...rules: PlanRule[]): void {
+    this.rules = [...this.rules, ...rules];
+  }
+
+  private renderPlanRules(rules: PlanRule[]): undefined | CfnBackupPlan.BackupRuleResourceTypeProperty[] {
+    if (!rules) {
+      return undefined;
+    }
+
+    const backupRuleResourceTypes: CfnBackupPlan.BackupRuleResourceTypeProperty[] = rules.map(rule => {
+      return {
+        ruleName: rule.ruleName,
+        targetBackupVault: rule.vault ? rule.vault.backupVaultName : this.vault.backupVaultName,
+        completionWindowMinutes: rule.completionWindow ? rule.completionWindow.toMinutes() : undefined,
+        startWindowMinutes: rule.startWindowAfter ? rule.startWindowAfter.toMinutes() : undefined,
+        lifecycle: this.parseRuleLifecycle(rule.lifecycle),
+        scheduleExpression: rule.schedule.expressionString
```

Comment:
Not 100% sure here but I think you have to validate that it's a `cron(...)` and not a `rate(...)`

Reply:
yes, if I also recall correctly, they only support cron expressions.